pkg/systemd/overview-cards/lastLogin.jsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2021 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React, { useState, useEffect } from 'react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { ExclamationTriangleIcon, UserIcon } from "@patternfly/react-icons";
+
+import cockpit from "cockpit";
+import { LoginMessage } from "./loginMessage.jsx";
+
+import './lastLogin.scss';
+
+const _ = cockpit.gettext;
+
+const LastLogin = () => {
+    const [messages, setLoginMessages] = useState(null);
+    const [username, setUsername] = useState(null);
+
+    useEffect(() => {
+        if (messages === null) {
+            const bridge = cockpit.dbus(null, { bus: "internal" });
+            bridge.call("/LoginMessages", "cockpit.LoginMessages", "Get", [])
+                    .then(reply => {
+                        const obj = JSON.parse(reply[0]);
+                        if (obj.version == 1) {
+                            if (obj['fail-count'] > 5) {
+                                obj.type = "danger";
+                            } else if (obj['fail-count'] > 0) {
+                                obj.type = "warning";
+                            } else {
+                                obj.type = "info";
+                            }
+
+                            setLoginMessages(obj);
+                        } else {
+                        // empty reply is okay -- older bridges just don't send that information
+                            if (obj.version !== undefined)
+                                console.error("unknown login-messages:", reply[0]);
+                        }
+                    })
+                    .catch(error => {
+                        console.error("failed to fetch login messages:", error);
+                    });
+        }
+        if (username === null) {
+            cockpit.user().then(user => setUsername(user.name));
+        }
+    }, [messages, username]);
+
+    if (messages === null || !messages['last-login-time'] || username === null) {
+        return null;
+    }
+
+    let icon = null;
+    let text = null;
+    const failedLogins = messages['fail-count'];
+
+    if (failedLogins) {
```

Comment:
So we no longer have orange and red versions?

Reply:
No, this is no longer in the design made by @garrett shown below.

Reply:
Implemented, that show a red icon when failed logins > 5.